test/widgets/content_test.dart

```diff
@@ -230,6 +230,48 @@ void main() {
     });
   });
 
+  group('ListNodeWidget', () {
+    testWidgets('ordered list with custom start', (tester) async {
+      await prepareContent(tester, plainContent('<ol start="3">\n<li>third</li>\n<li>fourth</li>\n</ol>'));
+
+      expect(find.text('3. '), findsOneWidget);
+      expect(find.text('4. '), findsOneWidget);
+      expect(find.text('third'), findsOneWidget);
+      expect(find.text('fourth'), findsOneWidget);
+    });
+
+    testWidgets('list uses correct text baseline alignment', (tester) async {
+      await prepareContent(tester, plainContent(ContentExample.orderedListLargeStart.html));
+
+      final table = tester.widget<Table>(find.byType(Table));
+      check(table.defaultVerticalAlignment).equals(TableCellVerticalAlignment.baseline);
+      check(table.textBaseline).equals(localizedTextBaseline(tester.element(find.byType(Table))));
+    });
+
+    testWidgets('ordered list markers have enough space to render completely', (tester) async {
+      await prepareContent(tester, plainContent(ContentExample.orderedListLargeStart.html));
+
+      final renderParagraph = tester.renderObject(find.textContaining('9999.')) as RenderParagraph;
+      final textHeight = renderParagraph.size.height;
+      final lineHeight = renderParagraph.text.style!.height! * renderParagraph.text.style!.fontSize!;
+      check(textHeight).equals(lineHeight);
+      check(renderParagraph.didExceedMaxLines).isFalse();
+    });
+
+    testWidgets('ordered list markers are end-aligned', (tester) async {
+      await prepareContent(tester, plainContent(ContentExample.orderedListLargeStart.html));
+
+      final marker9999 = tester.renderObject(find.textContaining('9999.')) as RenderParagraph;
+      final marker10000 = tester.renderObject(find.textContaining('10000.')) as RenderParagraph;
+      check(marker9999.size.width != marker10000.size.width).isTrue();
+
+      final marker9999Pos = marker9999.localToGlobal(Offset.zero);
+      final marker10000Pos = marker10000.localToGlobal(Offset.zero);
+      check(marker9999Pos.dx + marker9999.size.width)
+        .equals(marker10000Pos.dx + marker10000.size.width);
```

Comment:
Cool, this test is accurate, but can be simplified using `getRect`:
```dart
      final marker9999 = tester.getRect(find.textContaining('9999.'));
```
then `marker9999.width` and `marker9999.right`.

Reply:
(If you dig into the implementation of `getRect`, you'll see it's doing something similar to this test's logic.)